spec/controllers/grade_entry_forms_controller_spec.rb

```diff
@@ -566,4 +566,41 @@
       include_examples 'switch assignment tests'
     end
   end
+
+  describe '#populate_grade_table' do
+    before(:each) do
+      get_as role, :populate_grades_table, params: { course_id: course.id, id: grade_entry_form_with_data.id }
+    end
+    context 'an instructor' do
```

Comment:
add a test that makes sure they can see _all_ the student data (not just some number > 0)